requirements.txt

```diff
@@ -1,3 +1,2 @@
-dagster
-dagster-cloud
-requests
+./my-dagster-project
+dagster-cloud
```

Comment:
ah, this is why we need both?  Because it's not a package dependency, but it's a deployment dependency?

Reply:
right - that was a great call 🙏

Reply:
i realized when a project has multiple repos/locations, it's likely the top level wont have a setup.py